cloud/aws/providerconfig/v1alpha1/types.go

```diff
@@ -368,5 +368,8 @@ type Instance struct {
 	EBSOptimized *bool `json:"ebsOptimized"`
 
 	// The tags associated with the instance.
-	Tags map[string]string `json:"tag"`
+	Tags map[string]string `json:"tags"`
```

Comment:
I'm not sure if this wanted to be `json:"tags"` (which I'd set in my PR) or `json:"tag"` that someone elses PR had set. Given that the rest of the fields had their JSON field name set to the same as the struct field name, I opted for `tags`.

Reply:
+1 Thanks for catching that